index.js

```diff
@@ -9,10 +10,26 @@ function CoffeeScriptFilter (inputTree, options) {
   Filter.call(this, inputTree, options)
   options = options || {}
   this.bare = options.bare
+  this.options = options;
 }
 
 CoffeeScriptFilter.prototype.extensions = ['coffee', 'litcoffee', 'coffee.md']
 CoffeeScriptFilter.prototype.targetExtension = 'js'
+CoffeeScriptFilter.prototype.baseDir = function() {
+  return __dirname;
```

Comment:
If I put a `throw new Error('x')` statement in front of the `return`, the test suite still passes. It looks like this method never runs. Surely that can't be right?